### PR TITLE
Rubocop linter Issues fix

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

File: app/controllers/template_builder_controller.rb [hunk 1]
```diff
@@ -62,18 +62,16 @@ def create
...
       detail:            params[:detail],
       value:             template_builder.to_pretty_json,
-      user_id:           current_user.id,
+      user_id:           current_user.id
     )
 
     if @cf_template.save
       render text: I18n.t('cf_templates.msg.created')
+    elsif @cf_template.errors[:json]
+      render text: @cf_template.errors[:json], status: 500
```
Comment (skyarch-networks) [inline app/controllers/template_builder_controller.rb]: __[brakeman]__
* Unescaped model attribute rendered inline [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting/)

**Comment by Joeper214** [comment]
Please Review and Merge PR. @pocke   :bow:

**Comment by pocke** [comment]
redirect_to @root-beer

File: app/controllers/infrastructures_controller.rb [hunk 2]
```diff
@@ -371,14 +371,15 @@ def project_exist
...
     return if Project.exists?(id: params[:project_id])
 
-    if current_user.master?
-      if session[:client_id].present?
-        path = projects_path(client_id: session[:client_id])
+    path =
+      if current_user.master?
+        if session[:client_id].present?
+          projects_path(client_id: session[:client_id])
       else
```
Comment (root-beer) [inline app/controllers/infrastructures_controller.rb]: I don't know which `if`  that corresponds to at a glance.

ref: https://github.com/cookpad/styleguide/blob/master/ruby.en.md#control-structures

Comment (Joeper214) [inline app/controllers/infrastructures_controller.rb]: I think the indentation confuses you. but let me know if this is alright? 

```
path =
      if current_user.master?
        if session[:client_id].present?
          projects_path(client_id: session[:client_id])
        else
          clients_path
        end
      else
        projects_path
      end
```

Comment (root-beer) [inline app/controllers/infrastructures_controller.rb]: That is good! :+1:

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -33,11 +33,11 @@ def restore_locale
       return
     end
 
-
-    if I18n::available_locales.include?(lang.to_sym)
-      I18n.locale = lang
+    I18n.locale =
+      if I18n::available_locales.include?(lang.to_sym)
+        lang
     else
```
Comment (root-beer) [inline app/controllers/application_controller.rb]: I think that this is need to fix the indentation.

Comment (Joeper214) [inline app/controllers/application_controller.rb]: :+1:

File: app/controllers/infrastructures_controller.rb [hunk 4]
```diff
@@ -390,14 +391,13 @@ def infrastructure_exist
     return if Infrastructure.exists?(id: params[:id])
 
     msg = "Infrastructure \##{params[:id]} does not exist."
-    if session[:project_id].present?
-      path = infrastructures_path(project_id: session[:project_id])
+    path =
+      if session[:project_id].present?
+        infrastructures_path(project_id: session[:project_id])
+    elsif current_user.master?
```
Comment (root-beer) [inline app/controllers/infrastructures_controller.rb]: :warning: Indentation

File: app/controllers/nodes_controller.rb [hunk 5]
```diff
@@ -246,10 +246,11 @@ def edit_attributes
   def get_rules
     group_ids = params[:group_ids] || []
 
-    if group_ids.length > 0
-      rules_summary = @infra.ec2.describe_security_groups({group_ids: group_ids})
+    rules_summary =
+      if group_ids.length > 0
+        @infra.ec2.describe_security_groups({group_ids: group_ids})
     else
```
Comment (root-beer) [inline app/controllers/nodes_controller.rb]: :warning: Indentation

File: app/controllers/infrastructures_controller.rb (hunk 4 above)
Comment (Joeper214) [inline app/controllers/infrastructures_controller.rb]: :+1:

File: app/models/template_builder/parameter.rb [hunk 6]
```diff
@@ -12,10 +12,11 @@ class InvalidDataType < StandardError; end
   def initialize(name, property)
     raise InvalidDataType unless property.can_parameterize?
 
-    if property.data_type == String
-      @type = 'String'.freeze
+    @type =
+      if property.data_type == String
+        'String'.freeze
     else # Array
```
Comment (root-beer) [inline app/models/template_builder/parameter.rb]: :warning: Indentation

File: app/models/template_builder/resource.rb [hunk 7]
```diff
@@ -66,10 +66,11 @@ def set_properties(hash)
   def set_refs_params(hash)
     props = {}
     each_exist_props(hash) do |prop, val|
-      if val.nil?
-        props[prop.name] = "#{@name}#{prop.name}"
+      props[prop.name] =
+        if val.nil?
+          "#{@name}#{prop.name}"
       else
```
Comment (root-beer) [inline app/models/template_builder/resource.rb]: :warning: Indentation

File: app/models/zabbix.rb [hunk 8]
```diff
@@ -159,15 +159,16 @@ def update_mysql(infra, fqdn)
     key = "mysql.login"
     infra.resources.ec2.each do |r|
       item_infos = get_item_info(r.physical_id, key, "search")
-      if fqdn
-        host_key = "[" + fqdn + "]"
+      host_key =
+        if fqdn
+          "[" + fqdn + "]"
       else
```
Comment (root-beer) [inline app/models/zabbix.rb]: :warning: Indentation

**Comment by root-beer** [comment]
Do you mind if I fix the indentations?

**Comment by Joeper214** [comment]
Yeah. That's fine.

File: app/controllers/nodes_controller.rb [hunk 9]
```diff
@@ -247,12 +247,13 @@ def edit_attributes
   def get_rules
     group_ids = params[:group_ids] || []
 
-    if group_ids.length > 0
-      rules_summary = @infra.ec2.describe_security_groups({group_ids: group_ids})
-    else
-      rules_summary = @infra.ec2.describe_security_groups()
+    rules_summary =
+      if group_ids.length > 0
```
Comment (skyarch-networks) [inline app/controllers/nodes_controller.rb]: __[rubocop]__
* Style/ZeroLengthPredicate: Use `!empty?` instead of `length > 0`.

**Comment by skyarch-networks** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 

https://www.sideci.com/github_repositories/31754199/auto_correct/rubocop/pull_request/201/branch/rubocop-fix

**Comment by root-beer** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by root-beer** [comment]
I'm done.
Please Review and Merge PR. @Joeper214 :bow:

**Comment by Joeper214** [comment]
:+1: . I'll review now.

**Comment by Joeper214** [comment]
LGTM. Thanks. @root-beer

**Pull request merged**